services/keycloak/index.html

```diff
@@ -1,5 +1,45 @@
 ---
 layout: index
-title: Aerogear Keycloak
+title: Identity Management
 section: mobile-services
 ---
+
+<div class="row">
+    <div class="col-md-2">
+        <img style="width: 100px; height: 100px;" src="/img/keycloak.png" />
```

Comment:
Personally I would remove images and icons from the titles. Just my preference though

Reply:
The acceptance crit asked for the icon. Will leave it for now and others might choose to move it somewhere else

Reply:
Ok